Flaky test: org.keycloak.testsuite.oauth.OfflineTokenTest#offlineTokenBrowserFlowIdleTimeExpired

## org.keycloak.testsuite.oauth.OfflineTokenTest#offlineTokenBrowserFlowIdleTimeExpired
[Keycloak CI - Base IT (6)](https://github.com/keycloak/keycloak/actions/runs/5751595182/job/15591299142) / [Pull Request #22222](https://github.com/keycloak/keycloak/pull/22222)

```
java.lang.AssertionError: expected:<0> but was:<1>
	at org.junit.Assert.fail(Assert.java:89)
	at org.junit.Assert.failNotEquals(Assert.java:835)
	at org.junit.Assert.assertEquals(Assert.java:647)
	at org.junit.Assert.assertEquals(Assert.java:633)
        at org.keycloak.testsuite.oauth.OfflineTokenTest.testOfflineSessionExpiration(OfflineTokenTest.java:915)
        at org.keycloak.testsuite.oauth.OfflineTokenTest.offlineTokenBrowserFlowIdleTimeExpired(OfflineTokenTest.java:758)
...
```

--- a/common/src/main/java/org/keycloak/common/util/Time.java
+++ b/common/src/main/java/org/keycloak/common/util/Time.java
@@ -24,7 +24,7 @@ import java.util.Date;
  */
 public class Time {
 
-    private static int offset;
+    private static volatile int offset;
 
     /**
      * Returns current time in seconds adjusted by adding {@link #offset) seconds.

